Makefile

```diff
@@ -3,8 +3,8 @@
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
 # cloud.redhat.com/clowder-bundle:$VERSION and cloud.redhat.com/clowder-catalog:$VERSION.
-IMAGE_TAG_BASE ?= quay.io/cloudservices/clowder
-CLOWDER_BUILD_TAG ?= $(shell git rev-parse --short=8 HEAD)
+IMAGE_TAG_BASE ?= quay.io/redhat-user-workloads/hcm-eng-prod-tenant/clowder/clowder
```

Comment:
There are changes that need to be made below - we should consider removing some older `make targets` - we are deprecating the base image here, and that is used as part of the build below, we should also consider completely removing the base dockerfile, as it's now useless.